metisp/workflows/metis/metis_classification.py

```diff
@@ -286,3 +286,227 @@
 flux_tab_class = classification_rule("FLUXCAL_TAB",
                                      {"pro.catg": "FLUXCAL_TAB",
                                       })
+
+
+# ----- LM LSS Classifications -----
+
+# Slitloss files (TODO: Check the difference to STATIC ones - doubly defined??  Also check whether img or lss mode)
+lm_slitloss_class = classification_rule("LM_ADC_SLITLOSS_RAW",
```

Comment:
I suggest to use the same names whenever possible, because we already decided on names for the DRLD.

That is, we already decided that (e.g.) this data product should be called `LM_ADC_SLITLOSS_RAW`, so I suggest sticking to that decision and call it the EDPS class `lm_adc_slitloss_raw_class`. Etc. Changing the names mainly leads to confusion, even when the names would be better.